backend/src/yelp/yelp-read.ts

```diff
@@ -30,7 +30,9 @@ async function findBusinessesByCityAndCategory(cityName: string, categories: str
 
 export async function readBusinessDetails(businessId: string): Promise<any> {
     const route = getBusinessDetailsRoute(businessId)
-    return makeRequest(route)
+    const result = await makeRequest(route)
+    if(result.error?.code === 'ACCESS_LIMIT_REACHED') throw Error('Sorry friend. You\'ve reached your request limit on Yelp for today.')
```

Comment:
is bad to be specific about the limit of request?

Reply:
Sort of. If Yelp changes their limit, we don't have to come back and change the message if we're being vague. So it's more robust this way.